zerver/management/commands/update_stream_active_status.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from typing import Any
+
+from django.conf import settings
+from typing_extensions import override
+
+from zerver.lib.logging_util import log_to_file
+from zerver.lib.management import ZulipBaseCommand
+from zerver.lib.streams import check_update_all_streams_active_status
+
+## Logging setup ##
+logger = logging.getLogger(__name__)
+log_to_file(logger, settings.DIGEST_LOG_PATH)
+
+
+class Command(ZulipBaseCommand):
+    help = """For all streams, update the `is_recently_active` field to False if no message has been sent to the stream in the last 180 days."""
+
+    @override
+    def handle(self, *args: Any, **options: Any) -> None:
+        count = check_update_all_streams_active_status()
```

Comment:
So it looks like this value isn't calculated when importing data from a previous Zulip server.

Probably the `import_realm.py` code could call it?

Reply:
Done.